scripts/ps.py

```diff
@@ -17,17 +17,28 @@
 
 import psutil
 
+PROC_STATUSES_RAW = {
+    psutil.STATUS_RUNNING: "R",
+    psutil.STATUS_SLEEPING: "S",
+    psutil.STATUS_DISK_SLEEP: "D",
+    psutil.STATUS_STOPPED: "T",
+    psutil.STATUS_TRACING_STOP: "t",
+    psutil.STATUS_ZOMBIE: "Z",
+    psutil.STATUS_DEAD: "X",
+    psutil.STATUS_WAKING: "W"
+}
```

Comment:
I checked take man page of ps in linux for a reference.
Found nothing about the status above.
Could you give me some hints?